packages/ripple-binary-codec/package.json

```diff
@@ -23,7 +23,7 @@
     "build": "tsc -b && copyfiles ./src/enums/definitions.json ./dist/enums/",
     "clean": "rm -rf ./dist && rm -rf tsconfig.tsbuildinfo",
     "prepare": "npm run build && npm test",
-    "test": "jest",
+    "test": "jest --verbose false --silent=false --runInBand ./test/*.test.js",
```

Comment:
Why does this need to change?

Reply:
Was just adding consistency among all packages, happy to change if you want

Reply:
What's the difference between `--verbose false` and `--verbose=false`? (It seems we do `--silent=false`, and `--verbose false` but the docs don't note any difference between those two syntaxes)

Reply:
Bump on removing `--runInBand` (for better runtime)

Reply:
Fixed

Reply:
I don't think this change was pushed

Reply:
Pushed